Lottie.Droid/Properties/AssemblyInfo.cs

```diff
@@ -18,7 +18,7 @@
 // The form "{Major}.{Minor}.*" will automatically update the build and revision,
 // and "{Major}.{Minor}.{Build}.*" will update just the revision.
 
-[assembly: AssemblyVersion("1.5.0.0")]
+[assembly: AssemblyVersion("1.5.1.0")]
```

Comment:
I would rather not update the AssemblyVersion every time because this can be quite breaking for dependant libraries.